### DISCUSSION TITLE
从哪个日志中能看到当前ilogtail的采集和发送性能指标呐？

### DISCUSSION THREAD

**Discussion by onejimmyboy** [alibaba/ilogtail#1355] [Help]
从哪个日志中能看到当前ilogtail的采集和发送性能指标呐？

**Comment by 7y-9** [comment]
1.84版本可以看下 ./snapshot/ilogtail_status.LOG

**Comment by linrunqi08** [comment]
目前iLogtail自身的监控数据可以参考下文档：https://ilogtail.gitbook.io/ilogtail-docs/configuration/logging
iLogtail自身metrics正在重构，进展可以关注下：https://github.com/alibaba/ilogtail/discussions/716 目前指标数据通过Promethus API输出的部分还没有开发，如果有兴趣可以认领一下

**Comment by linrunqi08** [comment]
跟https://github.com/alibaba/ilogtail/issues/1325 重复了，后续可以关注下这个issue
